core/src/main/java/cz/o2/proxima/repository/AttributeDescriptor.java

```diff
@@ -54,9 +55,11 @@ private Builder(Repository repo) {
       Objects.requireNonNull(name, "Please specify name");
       Objects.requireNonNull(entity, "Please specify entity");
       Objects.requireNonNull(schemeURI, "Please specify scheme URI");
+
+      ValueSerializerFactory<?> factory = repo.getValueSerializerFactory(schemeURI.getScheme());
```

Comment:
This should be probably annotated @Nullable.

Reply:
Happy to. Can you enlighten me on what is a benefit of the @Nullable annotation in this particular case.

Reply:
It is a general design, that what can be assigned null value is annotated as @Nullable (although this general design might have some - unintended - exceptions :)). This annotation has benefits that {Find,Spot}Bugs will warn about usage of this field that is not checked against null value.

Reply:
Done